### PR TITLE
Combine insights components into InsightsPage

### PR DESCRIPTION
**Pull request by V-X-L-U**
In this PR:

- [x] Added the pie chart, bar chart, and date selectors into `InsightsPage`
- [x] Added translucent backdrop to the pie chart

Not in this PR:

- Linking code
- **The header of the `InsightsPage` will be added in the navigation menu PR**

### CODE REVIEW

File: client/src/components/TaskStatusChart/TaskStatusChart.js [hunk 1]
```diff
@@ -30,35 +30,38 @@ const chartLabel = ({
   );
 };
 
-const COLORS = ["#629632", "#F6BE00", "#FF0000"];
+const COLORS = ["#4F7009", "#F6BE00", "#cc0000"];
 
 export const TaskStatusChart = ({ tasksData, chartHeader }) => {
```
Comment (hiimchrislim) [inline client/src/components/TaskStatusChart/TaskStatusChart.js:34]: Minor comment: Is there a way to remove where it says 3 of 3?

image

Comment (hiimchrislim) [inline client/src/components/TaskStatusChart/TaskStatusChart.js:34]: Minor comment: Is there a way to remove where it says 3 of 3?

image

File: client/src/components/TaskStatusChart/TaskStatusChart.js [hunk 2]
```diff
@@ -30,35 +30,38 @@ const chartLabel = ({
   );
 };
 
-const COLORS = ["#629632", "#F6BE00", "#FF0000"];
+const COLORS = ["#4F7009", "#F6BE00", "#cc0000"];
```
Comment (hiimchrislim) [inline client/src/components/TaskStatusChart/TaskStatusChart.js]: I would move this constant inside the TaskStatusChart component

Comment (hiimchrislim) [inline client/src/components/TaskStatusChart/TaskStatusChart.js]: I would move this constant inside the TaskStatusChart component

File: client/src/components/TaskStatusChart/TaskStatusChart.js (hunk 1 above)
Comment (hiimchrislim) [reply client/src/components/TaskStatusChart/TaskStatusChart.js:34]: Perhaps you might also want to find a way to override the style for the background changing when you hover over it. Unless if you wanted to keep it as is

image

Comment (hiimchrislim) [reply client/src/components/TaskStatusChart/TaskStatusChart.js:35]: Perhaps you might also want to find a way to override the style for the background changing when you hover over it. Unless if you wanted to keep it as is

image

**Review by hiimchrislim — CHANGES_REQUESTED** [review]
Overall, it looks amazing! I left a few comments to take a look at. Afterwards, it should be good to merge!

If you have any questions, please let me know

**Comment by hiimchrislim** [comment]
You might want to also look into making the chart a bit larger as well. (Just a suggestion)
image
When compared to the pie chart, it looks a bit small

**Comment by hiimchrislim** [comment]
Also, please resolve the conflicts with the InsightsPage. Once the conflicts are resolved, you should double check that everything is working as intended

File: client/src/components/TaskStatusChart/TaskStatusChart.js (hunk 2 above)
Comment (Nestor-19) [reply client/src/components/TaskStatusChart/TaskStatusChart.js]: Resolved: I implemented this change.

File: client/src/components/TaskStatusChart/TaskStatusChart.js (hunk 1 above)
Comment (Nestor-19) [reply client/src/components/TaskStatusChart/TaskStatusChart.js:34]: Resolved: I used CSS to override this style.

Comment (Nestor-19) [reply client/src/components/TaskStatusChart/TaskStatusChart.js:34]: Resolved: I used CSS to override this style.

**Review by hiimchrislim — CHANGES_REQUESTED** [review]
Everything looks good! We just have to resolve the conflict with the latest develop branch and once those are fixed, we should be good to merge

**Comment by hiimchrislim** [comment]
We should make this portion a bit longer (to avoid the scrollbar showing when you pan the cursor from the right side of the chart)

image

**Comment by hiimchrislim** [comment]
The pie chart doesn't display the data for some reason (Show be yellow)

If possible, we should remove the 0% (2nd image)

image

image

**Review by hiimchrislim — APPROVED** [review]
LGTM!

**Pull request merged**